.golangci.yml

```diff
@@ -0,0 +1,148 @@
+linters-settings:
```

Comment:
What is the original source of this configuration file? Asking because there are some stuff I don't understand (like why it has gci configuration)